### PR TITLE
detect when user has completely losing position and subtract frozen f…

### PR DESCRIPTION
**Pull request by bthaile**
…unds of that position

 - when market is finalized and user has complete losing positions don't include frozen funds in display value in UI. 
 - added console logs for payment in dai and eth for debugging purposes. 
 - fixed flash script creating orders for scalar markets, bid and ask was backwards.

### CODE REVIEW

File: packages/augur-sdk/src/state/getter/Users.ts [hunk 1]
```diff
@@ -965,9 +965,29 @@ export class Users {
       _.values(_.mapValues(profitLossResultsByMarketAndOutcome, _.values))
     );
 
+    const marketFinalizedResults = await db.MarketFinalized.where('market')
+      .anyOf(_.map(allProfitLossResults, 'market'))
+      .toArray();
+    const marketFinalizedByMarket = _.keyBy(marketFinalizedResults, 'market');
+    const marketsData = await db.Markets.where('market')
+      .anyOf(_.map(allProfitLossResults, 'market'))
```
Comment (nuevoalex) [inline packages/augur-sdk/src/state/getter/Users.ts]: this could just be the smaller set of markets from `marketFinalizedResults` instead I think

Comment (bthaile) [reply packages/augur-sdk/src/state/getter/Users.ts]: ` .anyOf(_.map(allProfitLossResults, 'market'))` filter finalized market by profit and loss markets.
ah! I see need to use finalized markets instead. will change, 👍

**Review by nuevoalex — APPROVED** [review]
One suggestion otherwise LGTM

**Comment by bthaile** [comment]
change made, thx 👍

**Pull request merged**